src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -1068,5 +1071,20 @@ public sealed override bool Equals(Symbol other, TypeCompareKind compareKind)
                 nps.Equals(this, compareKind) :
                 base.Equals(other, compareKind);
         }
+
+#nullable enable
+        internal override UseSiteInfo<AssemblySymbol> GetUseSiteInfo()
```

Comment:
>GetUseSiteInfo [](http://example.com/codeflow?start=54&length=14)

It looks like we are adding this error to the declaring method by explicitly iterating through the parameters. I think there is no reason to override this method or cache information on this symbol.